images/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN make clean && \
 WORKDIR /tmp/ddptool
```

Comment:
I need to also add git on the `RUN apk add --no-cache --virtual build-dependencies build-base linux-headers` line. If I don't do that, I get an error after removing the version from alpine.